### PR TITLE
feat: Fontawesome 5.2.0

### PR DESCRIPTION
**Pull request by jcperez-ch**
Upgrade support to `fontawesome@5.2.0`

### CODE REVIEW

File: fontawesome/fontawesome.css [hunk 1]
```diff
@@ -3537,29 +3639,3 @@ readers do not read off random characters that represent icons */
   overflow: visible;
   position: static;
   width: auto; }
-@font-face {
```
Comment (rstoenescu) [inline fontawesome/fontawesome.css]: Hi, Thanks for doing this. But removing this and the following lines breaks fontawesome.

Comment (jcperez-ch) [reply fontawesome/fontawesome.css]: ok, I'll add them back and test accordingly...

**Comment by jcperez-ch** [comment]
@rstoenescu updated...

**Comment by rstoenescu** [comment]
Thank you for taking time on this PR! Releasing a new Quasar + CLI version in a few hours. This will be in it.

**Comment by jcperez-ch** [comment]
Thanks @rstoenescu, question, is this going to be released to `npm` soon? so I can take advantage of the upgrade?  Let me know ;)

**Comment by jcperez-ch** [comment]
I just saw the release! good good! 🎉 !!

**Pull request merged**